### PR TITLE
Upgrade CRA template to v5 of the iTwin Viewer

### PR DESCRIPTION
**Pull request by kckst8**

### CODE REVIEW

**Review by aruniverse — APPROVED** [review]

File: packages/modules/cra-template-itwin-viewer/template.json [hunk 1]
```diff
@@ -14,13 +14,13 @@
       "@bentley/imodeljs-markup": "^2.6.4",
       "@bentley/imodeljs-quantity": "^2.6.4",
       "@bentley/itwin-client": "^2.6.4",
-      "@bentley/itwin-viewer-react": "^4.0.0",
+      "@bentley/itwin-viewer-react": "^5.0.0",
```
Comment (calebmshafer) [inline packages/modules/cra-template-itwin-viewer/template.json:17]: Does this not link with `rush.json` to the current version of the itwin-viewer-react?

Comment (aruniverse) [reply packages/modules/cra-template-itwin-viewer/template.json:17]: Well isnt this because we're using independent versioning in the repo?

Comment (calebmshafer) [reply packages/modules/cra-template-itwin-viewer/template.json:17]: Ah yea, you're right.  I should have looked at it first :)

Comment (kckst8) [reply packages/modules/cra-template-itwin-viewer/template.json:17]: yes. for the time being, I'd prefer to verify major upgrades....at least until we can be more confident about semver across the board for our dependencies

**Review by Carl-Hinkle — DISMISSED** [review]

**Review by aruniverse — APPROVED** [review]

**Pull request merged**